lib/pretty.ml

```diff
@@ -11,7 +11,7 @@
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
```

Comment:
Let's delete this old pretty printer and replace any uses of it with the new one.